apps/kyc-server/src/components/ErrorBoundary.tsx

```
@@ -0,0 +1,56 @@
+import React, { ErrorInfo, ReactNode } from 'react'
```

Biome (2.1.2)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)